src/foam/core/types.js

```diff
@@ -106,6 +106,20 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
+  package: 'foam.core',
+  name: 'FormattedString',
+  extends: 'String',
+  documentation: 'A delimiter separated string of digits',
+
+  properties: [
+    {
+      name:'formatter',
```

Comment:
Document what format this value should be supplied as.

Reply:
@noodlemoodle perhaps an example.